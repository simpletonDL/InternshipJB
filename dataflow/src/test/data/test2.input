a = 1
b = a
x = 3
y = 4

while b < 10 {
  z = x
  b = b + 1
  x = 9
  y = 10
}