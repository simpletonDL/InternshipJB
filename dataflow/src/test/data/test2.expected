Assignment y = 4 not used in line 4
Assignment z = x not used in line 7
Assignment y = 10 not used in line 10
